docs/src/basic_concepts/record_fields.md

```diff
@@ -1,28 +1,22 @@
 ## Record organization
-The format uses various fields to organize and distinguish **records**
-and **record sets**[^1]:
-`name`, `uas_id` and `record_number`.
+The format uses different structures to organize and distinguish data.
 
 ### Records
-**Records** are used to store data both in **detection** as well as **tracking**.
-Each record must have a unique `record_number` inside **detection** or **tracking**,
-meaning that the same `record_number` can appear in both without causing conflict.
+**Records** represent all the information related with an entity in a given time and point in space.
```

Comment:
Point in space is optional in detection sets so say "in a given time and, when applicable, point in space" or something like that